src/System.ObjectModel/src/System/ComponentModel/INotifyPropertyChanged.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.ComponentModel
+{
+    public interface INotifyPropertyChanged
+    {
+        /// <devdoc>
```

Comment:
A bunch of these members have empty <devdoc/> attributes.  Should they just be removed?